image_view/scripts/extract_images_sync

```diff
@@ -80,6 +89,17 @@ Typical command-line usage:
         sync.registerCallback(self.save)
 
     def save(self, *imgmsgs):
+        delay = self.get_clock().now() - self.time
+
+        # Decimation is enabled if it is set to greater than
+        # zero. If it is enabled, exit if delay is less than
+        # the decimation rate specified.
+        if (self.sec_per_frame > 0) and (delay < self.sec_per_frame):
```

Comment:
you can't directly compare a Duration and a float - need to do a conversion here:
```
  File "/home/fergs/image/install/image_view/lib/image_view/extract_images_sync", line 97, in save
    if (self.sec_per_frame > 0) and (delay < self.sec_per_frame):
TypeError: '<' not supported between instances of 'Duration' and 'float'
```